src/components/settings/Avatar.svelte

```diff
@@ -35,6 +35,6 @@
   <div
     class="flex items-center justify-center bg-base-content text-base-100 uppercase font-bold rounded-full {sizeClasses}"
   >
-    {$sessionStore.username[0]}
+    {$sessionStore.username.trimmed[0]}
```

Comment:
> I think this will only return the first character.

that's the idea :) when a user hasn't set an avatar, i just grab the first character from their username and use it to make the avatar 
<img width="485" alt="image" src="https://user-images.githubusercontent.com/1179291/206005259-00b7a059-e8eb-4a89-ab41-028f8e8f6395.png">